e3/archive.py

```diff
@@ -44,12 +44,14 @@ def __select_archiving_tool(filename, unpack=True,
     :return: a tuple (ext, True if we should use python libraries else False)
     :rtype: (str, bool)
     """
+
     def has_python_lib_support(archive_ext):
```

Comment:
No need anymore for has_python_lib. We assume that we have a full python and thus tarfile and zipfile are always there